code/datums/spells/mime_malaise.dm

```diff
@@ -38,16 +38,26 @@
 		unEquip(wear_mask, TRUE)
 		unEquip(w_uniform, TRUE)
 		unEquip(wear_suit, TRUE)
+		if(is_species(src, /datum/species/plasmaman))
+			unEquip(head, TRUE)
+			equip_to_slot_if_possible(new /obj/item/clothing/head/helmet/space/plasmaman/mime, slot_head, TRUE, TRUE)
+			equip_to_slot_if_possible(new /obj/item/clothing/under/plasmaman/mime, slot_w_uniform, TRUE, TRUE)
+		else
+			equip_to_slot_if_possible(new /obj/item/clothing/under/mime, slot_w_uniform, TRUE, TRUE)
 		equip_to_slot_if_possible(new /obj/item/clothing/mask/gas/mime, slot_wear_mask, TRUE, TRUE)
-		equip_to_slot_if_possible(new /obj/item/clothing/under/mime, slot_w_uniform, TRUE, TRUE)
 		equip_to_slot_if_possible(new /obj/item/clothing/suit/suspenders, slot_wear_suit, TRUE, TRUE)
 		Silence(7)
 	else
 		qdel(wear_mask)
 		qdel(w_uniform)
 		qdel(wear_suit)
+		if(src.dna.species.name == "Plasmaman")
```

Comment:
😠

Reply:
Most recent commit should've fixed this. Not sure why it's still showing up in this format.

Reply:
Oh, I'm dumbass.